Sources/Components/BroadcastContainerView/BroadcastContainerView.swift

```diff
@@ -21,6 +21,20 @@ public protocol BroadcastContainerViewDelegate: class {
     ///   - newContainerSize: the size that the BroadcastContainerView will have when the broadcast is dismissed
     ///   - commitToDismissal: a closure that must be called in order for the BroadcastContainerView to dismiss the broadcast
     func broadcastContainer(_ broadcastContainerView: BroadcastContainerView, willDismissBroadcastAtIndex index: Int, withNewContainerSize newContainerSize: CGSize, commitToDismissal: @escaping (() -> Void))
+
+    /// Called when a URL in a broadcasts message is tapped
+    ///
+    /// - Parameters:
+    ///   - broadcastContainerView: the BroadcastContainerView
+    ///   - url: the URL that was tapped
+    ///   - index: the index of the broadcast that containes the URL that was tapped
+    func broadcastContainer(_ broadcastContainerView: BroadcastContainerView, urlTapped url: URL, inBroadcastAtIndex index: Int)
```

Comment:
Why not `dudTapURL url: URL atIndex index: Int`?

Reply:
Agree 👍